src/ru/nsu/ccfit/zuev/osu/scoring/ScoringScene.java

```diff
@@ -218,6 +231,7 @@ public void load(final StatisticV2 stat, final BeatmapInfo beatmap,
                     new FadeInModifier(2), new ScaleModifier(2, 2, 1)));
         }
         mark.setPosition((float) (Config.getRES_WIDTH() * 5) / 6 - mark.getWidth() / 2, 80);
+        scene.attachChild(mark);
```

Comment:
Is there a reason why this isn't attached to `scaledContainer`? This is a skinnable element, and it may overlap if padding is present.